docs/gke/configs/cluster.jinja

```diff
@@ -72,59 +83,85 @@ resources:
       monitoringService: monitoring.googleapis.com/kubernetes
       {% else %}
       initialClusterVersion: 1.9.6-gke.1
-      {% endif %}      
+      {% endif %}
+      podSecurityPolicyConfig:
```

Comment:
Should we make all the security parameters fields in a dictionary "securityFeatures" just to make the properties a little bit more human readable?